mm-mock-server/src/mocks/queries.ts

```diff
@@ -22,20 +22,10 @@ export function mockQueries(serverState: MockServerState) {
             return serverState.otherUsers.concat([serverState.loggedInUser]);
         },
         getAuthenticatedUser: () => {
-            if (!serverState.loggedIn) {
-                return null;
-            }
```

Comment:
I believe the real backend returns null when the user is not logged in, aside from throwing an exception. From what Holger mentioned I believe this means that now we can always expect a User and we will instead verify some other field (or more specific exception) to determine if the user is authorized.

Reply:
You're right, didn't realize that the backend was returning `data: null` on errors.

```json
{"errors":[{"message":"Cannot return null for non-nullable field Query.getAuthenticatedUser.","locations":[{"line":2,"column":3}],"path":["getAuthenticatedUser"],"extensions":{"code":"INTERNAL_SERVER_ERROR","stacktrace":["Error: Cannot return null for non-nullable field Query.getAuthenticatedUser.","    at completeValue (/Users/chris/Documents/mm3/mmdata/node_modules/graphql/execution/execute.js:594:13)","    at /Users/chris/Documents/mm3/mmdata/node_modules/graphql/execution/execute.js:486:9","    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)","    at async Promise.all (index 0)","    at async execute (file:///Users/chris/Documents/mm3/mmdata/node_modules/@apollo/server/dist/esm/requestPipeline.js:220:37)","    at async processGraphQLRequest (file:///Users/chris/Documents/mm3/mmdata/node_modules/@apollo/server/dist/esm/requestPipeline.js:155:32)","    at async internalExecuteOperation (file:///Users/chris/Documents/mm3/mmdata/node_modules/@apollo/server/dist/esm/ApolloServer.js:585:16)","    at async runHttpQuery (file:///Users/chris/Documents/mm3/mmdata/node_modules/@apollo/server/dist/esm/runHttpQuery.js:129:29)","    at async runPotentiallyBatchedHttpQuery (file:///Users/chris/Documents/mm3/mmdata/node_modules/@apollo/server/dist/esm/httpBatching.js:34:16)","    at async ApolloServer.executeHTTPGraphQLRequest (file:///Users/chris/Documents/mm3/mmdata/node_modules/@apollo/server/dist/esm/ApolloServer.js:496:20)"]}}],"data":null}
```